ee-feature-pack/galleon-feature-pack/wildfly-feature-pack-build.xml

```diff
@@ -125,7 +125,6 @@
                 <extension>org.wildfly.extension.messaging-activemq</extension>
                 <extension>org.wildfly.extension.microprofile.config-smallrye</extension>
                 <extension>org.wildfly.extension.microprofile.health-smallrye</extension>
-                <extension>org.wildfly.extension.microprofile.metrics-smallrye</extension>
```

Comment:
Is the new extension not being added here because org.wildfly.extension.microprofile.metrics-smallrye shouldn't have been in the domain section in the first place?